Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/PlayerMovedMessage.java

```diff
@@ -18,20 +20,42 @@
     private final HandbrakeAction handbrakeAction;
     private final ThrottleAction throttleAction;
 
+    /**
+     * Initializes the PlayerMovedMessage class.
+     *
+     * @param steerAction The steer action of the player, not null.
+     * @param handbrakeAction The handbrake action of the player, not null.
+     * @param throttleAction The throttle action of the player, not null.
+     */
     public PlayerMovedMessage(SteerAction steerAction, HandbrakeAction handbrakeAction, ThrottleAction throttleAction) {
         this.steerAction = steerAction;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)